samples/nrf9160/nrf_cloud_rest_agps/README.rst

```diff
@@ -0,0 +1,351 @@
+
+* `Ephemeris`_
+* `Almanac`_
+* UTC time
+* Integrity data
+* Approximate location
+* GPS system clock
+* Ionospheric correction parameters for `Klobuchar Ionospheric Model`_
+
+.. note::
+
+   .. include:: ../../../doc/nrf/libraries/networking/nrf_cloud_agps.rst
+      :start-after: agpslimitation_start
+      :end-before: agpslimitation_end
+
+TTFF is expected to be less when compared to that in the scenario of GPS without assistance data, performing a cold or a warm start.
+The TTFF is expected to be comparable to that of a GPS hot start and is usually in the range of 5 to 20 seconds in good satellite signaling conditions.
+Good satellite signaling conditions in this context means that the device is positioned in a way to have a direct line of sight to a large portion of the sky.
+
+Note that the radio component on the nRF9160-based device is shared between LTE and GPS and hence only one of these can be active at any moment of time.
+LTE has priority over the radio resources, and GPS operates in the time interval between LTE required operations as per LTE specification.
+Consequently, GPS is blocked from searching as long as the modem is in `RRC connected mode <RRC idle mode_>`_.
+
+The modem is in RRC connected mode when sending and receiving data over the LTE network.
+After data transmission is complete, the modem stays in RRC connected mode for a period of network-specific inactivity time, before entering RRC idle mode.
+The inactivity time varies between networks.
+It is typically in the range of 5 to 70 seconds for a specific network.
+In RRC idle mode, the GPS is usually able to operate.
+
+It is recommended to use LTE Power Saving Mode (PSM) and extended Discontinuous Reception (eDRX) mode to increase the allowed time of operation for GPS.
+During the defined intervals of PSM and eDRX, LTE communication does not occur, and the GPS has full access to the radio resources.
+In this sample, both PSM and eDRX are enabled by default.
+You can enable or disable these features by using the :kconfig:`CONFIG_LTE_POWER_SAVING_MODE` and :kconfig:`CONFIG_LTE_EDRX_REQ` configuration options.
+
+.. include:: /applications/asset_tracker/README.rst
+   :start-after: external_antenna_note_start
+   :end-before: external_antenna_note_end
+
+User interface
+**************
+
+If :kconfig:`CONFIG_REST_DO_JITP` is enabled, pressing button 1 will request just-in-time provisioning with nRF Cloud.
+This is useful when initially provisioning and associating a device on nRF Cloud.
+You only need to do this once.
+
+If :kconfig:`CONFIG_REST_ENABLE_LED` is enabled, LED1 indicates the state of the connection to nRF Cloud.
+
+Configuration
+*************
+|config|
+
+Configuration options
+=====================
+
+Check and configure the following configuration options for the sample:
+
+.. option:: CONFIG_LTE_POWER_SAVING_MODE - LTE Power Saving Mode
+
+This configuration option enables or disables the LTE Power Saving Mode.
```

Comment:
```suggestion
.. kconfig:: CONFIG_LTE_POWER_SAVING_MODE

CONFIG_LTE_POWER_SAVING_MODE - LTE Power Saving Mode
   This configuration option enables or disables the LTE Power Saving Mode.
```

Reply:
If you decide **not** to add this option to the Kconfig file, please remove this and the reference to it on line 66.

Reply:
I was going to comment about this, but seems that I forgot to do so. The name is a bit confusing, because for example CONFIG_LTE_EDRX_REQ is a configuration option for LTE LC, but CONFIG_LTE_POWER_SAVING_MODE is not. But the same naming is used in some other sample(s) as well. There's no Kconfig flag in LTE LC for PSM, but this needs to be in the application Kconfig and the application needs to call `lte_lc_psm_req()` function to enable PSM.

Reply:
I copied the readme from the agps sample and edited it, and did not notice that this kconfig was there but not in the code.  I will add it to the code to enable PSM mode, similar to how it is done in the agps sample.